lib/EmitVerilog/EmitVerilog.cpp

```diff
@@ -1581,6 +1583,37 @@ void ModuleEmitter::emitDecl(InstanceOp op) {
   indent() << ");\n";
 }
 
+void ModuleEmitter::emitStatement(rtl::RTLInstanceOp op) {
+  SmallPtrSet<Operation *, 8> ops;
+  ops.insert(op);
+
+  auto instanceName = op.instanceName();
+  StringRef defName = op.moduleName();
+
+  auto opArgs = op.inputs();
+
+  auto moduleIR = op.getParentOfType<firrtl::CircuitOp>();
+  auto referencedModule =
+      cast<rtl::RTLModuleOp>(moduleIR.lookupSymbol(defName));
```

Comment:
I think this will crash if you use a RTL instance with FIRRTL module.  Does the verifier correctly reject this?